app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala

```diff
@@ -111,7 +126,18 @@ class DesConnector @Inject() (
       val encodedClientId = UriEncoding.encodePathSegment(mtdItId.value, "UTF-8")
       val url = new URL(s"$baseUrl/registration/relationship?ref-no=$encodedClientId&agent=false&active-only=true&regime=ITSA")
 
-      getWithDesHeaders[RelationshipResponse]("GetStatusAgentRelationship", url).map(_.relationship.headOption).recover {
+      getWithDesHeaders[ItsaRelationshipResponse]("GetStatusAgentRelationship", url).map(_.relationship.headOption).recover {
+        case e: NotFoundException => None
+      }
+    }
+  }
+
+  def getActiveClientVatRelationships(vrn: Vrn)(implicit c: HeaderCarrier, ec: ExecutionContext): Future[Option[VatRelationship]] = {
+    monitor(s"ConsumedAPI-Get-VAT-Relationship-GET") {
```

Comment:
`ConsumedAPI-GetActiveClientVatRelationships-GET`, similar change for `getActiveClientItsaRelationships`